### PR TITLE
UpdateYourYearPLZ

### PR DESCRIPTION
**Pull request by jusleg**
One should always make sure that the year in the footer is up to date. Last I checked the year is still valid. In a coupled of days it will not. We gotta be ready for when the ball drops. 

ball drop

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,6 +184,7 @@ <h3 class="center-align white-text">Exam <strong>Bank</strong><span id="emoji-ca
 		<script async src="script/analytics.js"></script>
         <script src="script/script.js"></script>
         <script src="https://cdn.jsdelivr.net/gh/kcamcam/emoji-calendar/emoji-calendar.js"></script>
+	<script src="https://cdn.rawgit.com/jusleg/updateYourYearPLZ/111cb805/updateYourYearPLZ.js"></script>
```
Comment (kcamcam) [inline index.html]: It looks like `rawgit` is going to be deprecated soon, can you serve update your please over another CDN such as `jsdelivr.net`?

**Comment by jusleg** [comment]
@kcamcam we should merge it before the ball drop. I added the new cdn link and did some formatting of the html

**Comment by kcamcam** [comment]
@jusleg the html seems to be broken, maybe there is a closing tag missing somewhere?

**Comment by kcamcam** [comment]
screen shot 2019-01-02 at 5 34 47 pm

**Comment by jusleg** [comment]
Damn damn I'll look into it

**Pull request merged**